common/src/test/java/com/tc/net/basic/BasicConnectionTest.java

```diff
@@ -112,6 +113,7 @@ public void testGetConnectTime() throws Exception {
    * Test of getIdleTime method, of class BasicConnection.
    */
   @Test
+  @Ignore
```

Comment:
This test does not mock things properly and can fail even on master and get caught in infinite loop causing out of memory problems.

Reply:
Can we fox the test instead?

Reply:
I will try and fix this in this PR itself